backend/src/tiedostot/ProjektiTiedostoManager/TiedostoManager.ts

```diff
@@ -40,7 +50,7 @@ export abstract class TiedostoManager<T> {
   // nähtävillä, kun tarkastellaan tuota parametrina annettuna vaihetta kutsumalla this.vaihe.
   abstract getAineistot(vaihe: T): AineistoPathsPair[];
 
-  abstract getLadatutTiedostot(vaihe: T): LadattuTiedosto[];
+  abstract getLadatutTiedostot(vaihe: T): LadattuTiedosto[][];
```

Comment:
Tää piti muuttaa tähän muotoon, jossa meidän handleChangedTiedostot-taikafunktio pystyy toimimaan oikein. Ts. tän pitää palauttaa LadatutTiedostot siten, että this.vaihe referoi yhä tasan samoihin tiedostoihin, jotka tää getLadatutTiedosto palauttaa. Jos eri LausuntoPyynnonTäydennyksien muistutukset laittaa concat:lla yhteen putkeen ja palauttaa silleen, toi handleChangedTiedostot ei enää toimi, koska vaikka se muokkaisia niitä concatilal yhteen arrayhyn laitettuja tiedostoja, vaikutukset ei heijastuisi this.vaihe:een sisältöön.